packages/squiggle-lang/src/ast/peggyParser.peggy

```diff
@@ -146,7 +146,7 @@ relational
   relationalOp "operator" = '<=' / '<' / '>=' / '>'
 
 credibleInterval
-  = head:additive tail:(__ @credibleIntervalOp __nl @additive)*
+  = head:additive tail:(__nl @credibleIntervalOp __nl @additive)*
```

Comment:
This is now allowed:
```
2
to
3
```

(I couldn't find how it could be ambiguous; this allowed me to delete `__` rule)